spec/features/editor_spec.rb

```diff
@@ -93,6 +94,22 @@
     end
   end
 
+  context 'when an exercise has one or more teacher-defined assessments' do
+    it 'displays the score button' do
+      visit(implement_exercise_path(exercise))
+      expect(page).to have_content(exercise.title)
+      expect(page).to have_css('#assess')
```

Comment:
We probably want to check for a more meaningful (semantic) attribute here. This is a bit brittle. But fine for now, I guess.

Reply:
That's true, what do you think if I swap it to the following? This would allow to check for the behaviour (having the test to trigger the action) instead of the specific implementation with the element ID.
```suggestion
      expect(page).to have_content(I18n.t('exercises.editor.score'))
```

Reply:
Better 👍

Reply:
Thanks, I've done so 🙂